background/redux-slices/0x-swap.ts

```diff
@@ -133,11 +134,23 @@ const get0xApiBase = (network: EVMNetwork) => {
   return `${prefix}${base}`
 }
 
-const gatedParameters = {
-  affiliateAddress: COMMUNITY_MULTISIG_ADDRESS,
-  feeRecipient: COMMUNITY_MULTISIG_ADDRESS,
-  buyTokenPercentageFee: SWAP_FEE,
+const getMultisigAddressByNetwork = (network: EVMNetwork) => {
```

Comment:
```suggestion
function getMultisigAddressByNetwork(network: EVMNetwork) {
```

Let's stick to regular function declarations unless we need the variable-assignment version for specific reasons (e.g. `this` binding, etc). Amongst other things, it gives us better stack traces (as functions attach their names to the stack in a way that `const`s don't).